src/cpu/x64/jit_brgemm_transpose_utils.hpp

```diff
@@ -145,10 +145,10 @@ struct jit_brgemm_trans_to_vnni_t {
         dim_t current_col_size, current_row_size;
     };
 
-    typedef enum matrix_to_transform {
+    enum class matrix_to_transform_t {
         matrix_B,
-        matrix_C
-    } matrix_to_transform_t;
+        matrix_C,
```

Comment:
Similar comment, naming is redundant.

Reply:
Same story, there's a connection with names from brgemm space that are not affected by tidy, so I'll let CPU team to deal with it, too.